src/Form.js

```diff
@@ -0,0 +1,102 @@
+import {useState} from 'react'
+import Button from './Button'
+import Field from './Field'
+
+function Form(){
+    const [isValid, setIsValid] = useState(false);
+    const [isEmailValid, setIsEmailValid] = useState(false)
+    const [hasLowerCase, setHasLowerCase] = useState(false);
+    const [hasUppercase, setHasUppercase] = useState(false);
+    const [hasNumber, setHasNumber] = useState(false);
+    const [hasSpecialChar, setHasSpecialChar] = useState(false);
+    const [hasMinEightChars, sethasMinEightChars] = useState(false);
+
+    function FormValidators() {
+      return (        
+        <div className='form-validators__wrapper'>
+          <div className='form-validators__group'>
+            <ul>
+              <li className={hasLowerCase ? 'form-validators__criteria__met' : ''}>One lowercase character</li>
+              <li className={hasUppercase ? 'form-validators__criteria__met' : ''}>One uppercase character</li>
+              <li className={hasNumber ? 'form-validators__criteria__met' : ''}>One number</li>
+            </ul>
+          </div>
+          <div className='form-validators__group'>
+          <ul>
+              <li className={hasSpecialChar ? 'form-validators__criteria__met' : ''}>One special character</li>
+              <li className={hasMinEightChars ? 'form-validators__criteria__met' : ''}>8 characters minimum</li>
+            </ul>
+          </div>
+        </div>
+        
+      )
+    }
+
+    function CorrectPasswordFeedback() {
+      return (
+      <div className='form-validators__feedback__success'>
+        Your password is secure and you're all set!
+        </div>
+      )
+    }
+
+    const handleChange = (event) =>  {
+    if(/.*\d.*/.test(event.target.value)) {
```

Comment:
Some indentation problem. Usually things like this are handled by tools like `prettier` or inside your IDE/Editor settings.

Reply:
These if statements are fine, but you can also minify it to e.g. 

`setHasNumber(/.*\d.*/.test(event.target.value))`